Adamant/Modules/Wallets/Dash/DashWalletService.swift

```diff
@@ -605,6 +585,15 @@ extension DashWalletService {
     }
 }
 
+#if DEBUG
+extension DashWalletService {
```

Comment:
Could we move this into the test target instead? Even though it is marked with DEBUG, for better organization it is better to put the code related to tests under test targets only. Would it be possible @ChrisBenua?

Reply:
Unfortunately, you can't access private properties of class outside of class declaration file. So there are only two options: make wallet property internal or add this extension for debug build only